### ISSUE TITLE
Symbolc conflict in Battlestar Galactica.

### ISSUE THREAD

**Issue by jarupxx** [Cxbx-Reloaded/Cxbx-Reloaded#94]
```
[0x4C90] HLE: 0x000E7560 -> XTL::EmuIDirect3DDevice8_SetPalette
[0x4C90] HLE: 0x000E7560 -> XTL::EmuIDirect3DDevice8_SetIndices
```

This is with Cxbx-Reloaded 7a4fca5bda7e21d3e0629c4f4fb10c74bec6e86e. It crashes due to false detection of SetIndices. This symbol is not detected in Dxbx.

**Comment by PatrickvL** [comment]
Good find! It's all those little things that, once fixed, will help us run more software, so thanks for this report.

Keep an eye out for more, and if possible, try to submit or suggest a fix for this (and any other issue you find). Cheers!

**Comment by PatrickvL** [comment]
PS: Do realise this issue applies to the current HLE, not the future LLE GPU emulation. But for now, HLE is the only path that works (somewhat), so it's alright to improve HLE (at least until LLE has matured enough to become the default).

**Comment by LukeUsher** [comment]
Thanks, it would help if you could tell us the version of XDK/D3D that this title uses, this can be found either in the kernel log, or by making an XBE dump [Edit => Dump XBE Info]

**Comment by jarupxx** [comment]
Hi! It is XDK 5558. Please see this file.
https://github.com/jarupxx/cxbx_log/blob/master/BattleStar%20Xbe.txt

**Comment by PatrickvL** [comment]
The root cause for this is that the D3D version 5558 OOVPA database also contains symbols for other versions (for example, D3DDevice_SetIndices_1_0_5455) which leads to conflicting detections.

When D3DDevice_SetIndices_1_0_5455 and IDirect3DDevice8_SetVertexData2f_1_0_4627 are commented out in OOVPATable D3D8_1_0_5558, Battlestar Galactica goes into menu's again.

**Comment by PatrickvL** [comment]
This issue can be closed once PR #102 is merged.

Even so, it would be good if Cxbx detects conflicts like these, and at least logs a message, or possibly show a message and/or stop emulation.

**Comment by LukeUsher** [comment]
I'll close this and create a new issue
